src/routes/api-documentation.ts

```diff
@@ -0,0 +1,38 @@
+import Router from '@koa/router';
+import { Context, Middleware, Next } from 'koa';
+import { hydra, owl, schema } from 'rdf-namespaces';
+import url from 'url';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.body = {
+      '@id': url.resolve(request.origin, router.url(Routes.ApiDocumentation, {})),
+      '@type': hydra.ApiDocumentation,
+      [hydra.entrypoint]: {
+        '@id': url.resolve(request.origin, router.url(Routes.EntryPoint, {})),
+      },
+      [hydra.supportedClass]: [
+        {
+          '@id': schema.EntryPoint,
+          '@type': hydra.Class,
+          [hydra.title]: 'API entry point',
+          [hydra.supportedOperation]: {
+            '@type': hydra.Operation,
+            [hydra.title]: 'Get the entry point',
+            [hydra.method]: 'GET',
+            [hydra.expects]: {
+              '@id': owl.Nothing,
+            },
+            [hydra.returns]: {
+              '@id': schema.EntryPoint,
+            },
+          },
```

Comment:
I haven't defined any `hydra:supportedProperty`, not sure there's much need? We'll be willing to pass a lot of unknown properties around, so feels like it's only worth declaring required (eg `schema:name`) and impossible (eg `schema:potentialActions`) properties. But then a generic Hydra UI can't produce a useful form...